exporter/lokiexporter/README.md

```diff
@@ -15,9 +15,203 @@ Exports data via HTTP to [Loki](https://grafana.com/docs/loki/latest/).
+```
+
+#### From `| json | an_attribute=...` to `{an_attribute=...}` or `| an_attribute=...`
+
+OTel log attributes, resource attributes, and fields are no longer stored in the JSON message but are stored as 
+* Loki message labels for promoted resource attributes (see list above),
+* Loki message metadata for other resource attributes, log attributes and log fields.
+
+LogQL statements `| json | an_attribute=...` must be converted to:
+
+* Promoted resource attributes: `{an_attribute=...}`
+* For other resource attributes, log attributes, and log fields: `| an_attribute=...`
+
+#### From `| json | traceid=...` and `| json | spanid=...` to `| trace_id=...` and `| span_id=...`
+
+The log fields `SpanID` and `TraceId` where stored as JSON field `spanid` and `traceid`; they are now stored as `metadata[span_id]` and `metadata[trace_id]`, LogQL queries must be changed accordingly.
+
+`TraceID` filters like `| json | traceid=<<traceId>> ...` and `|= <<traceId>> ...` must be converted to `| trace_id=<<traceId>> ...` where `<<traceId>>` and <<spanId>> are the values you search for.  
+Similarly, `SpanID` filters like `| json | spanid=<<spanid>> ...` and `|=<<spanid>> ...` must be converted to `| span_id=<<spanid>> ...`.
+
+Example:
+
+```
+BEFORE
+{exporter="OTLP", job="ecommerce/frontend"} | json | resources_deployment_environment="production" |= "00960a472ea5b87954ca07902d66f914" ...
+
+AFTER
+{service_namespace="ecommerce", service_name="frontend"} | deployment_environment="production" | trace_id="00960a472ea5b87954ca07902d66f914"...
+```
+
+#### From `line_format {{.body}}` to `line_format {{__line__}}`
+
+The `{{.body}}` element of the JSON payload that used to hold the OTel log message body is now the message of the Loki log line and should be referenced as `{{__line__}}` in `line_format` calls.
+
+Example:
+
+```
+AFTER
+{service_namespace="ecommerce", service_name="frontend"} | deployment_environment =~ `production` | line_format `[{{.detected_level}}] {{__line__}}`
+```
+
+#### Using `keep __line__` in direct LogQL queries to prevent stream splits
+
+The number of logstreams return by LogQL queries made directly to Loki (without going through the Grafana GUI) may increase due to the distinct metadata label values and thus impact the sorting of the result as results are sorted per log stream.
+Use the `keep` function to limit the number of log streams return by LogQL queries and have a global doring of the returned  log entries
+
+Example:
+
+```
+AFTER
+... | keep __line__, detected_level
+```
+
+### Grafana visualizations migration
+
+#### Tempo data source: Trace 2 Logs
+
+To enable the "trace to logs" navigation from Tempo to Loki, navigate to the Grafana Tempo data source configuration screen, in the "Trace to logs" section, select "use custom query" and specify the query:
+
+```
+{service_name="${__span.tags["service.name"]}"} | trace_id="${__span.traceId}"
+```
+
+#### Loki data source: Log 2 Trace
```

Comment:
```suggestion
#### Loki data source: Log to Trace
```
I can't find any reference to `2` being included in documentation, but let me know if I missed something.